File: app/src/main/java/ua/edu/deanoffice/mobile/studentchdtu/service/pojo/ApplicationTypePOJO.java
package ua.edu.deanoffice.mobile.studentchdtu.service.pojo;

public class ApplicationTypePOJO {
    private int id;
    public String name;

    public ApplicationTypeIdPOJO toApplicationTypeId() {
        return new ApplicationTypeIdPOJO(id);
    }

}

File: app/src/main/java/ua/edu/chdtu/deanoffice/mobile/service/POJO/ApplicationTypePOJO.kt
package ua.edu.chdtu.deanoffice.mobile.service.POJO

data class ApplicationTypePOJO(
    private val id: Int,
    val name: String
    ){

    fun toApplicationTypeId() : ApplicationTypeIdPOJO{
        return ApplicationTypeIdPOJO(id)
    }
}